app/renderer/components/main/siteInfo.js

```diff
@@ -248,11 +243,11 @@ class SiteInfo extends React.Component {
   render () {
     return <Dialog testId='siteInfoDialog' onHide={this.onHide} className='siteInfo' isClickDismiss>
       <div onClick={(e) => e.stopPropagation()}
-        className={cx({
-          [css(commonStyles.flyoutDialog)]: true,
-          [css(styles.siteInfo__wrapper)]: true,
-          [css(styles.siteInfo__wrapper__large)]: (this.props.isBlockedRunInsecureContent || this.props.runInsecureContent)
-        })}>
+        className={css(
+          commonStyles.flyoutDialog,
```

Comment:
Creating `<FlyoutDialog>` to remove commonStyles.flyoutDialog.

Reply:
there's a PR for that from you (#10978) should it land first then this?

Reply:
It is ok to merge this first.